com.vbam.visualboyadvancem.yaml

```diff
@@ -0,0 +1,119 @@
+app-id: com.vbam.visualboyadvancem
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: visualboyadvance-m
+rename-icon: visualboyadvance-m
+rename-desktop-file: visualboyadvance-m.desktop
+finish-args:
+  - --device=all
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=host:ro
```

Comment:
why host filesystem access?

Reply:
How is is someone going to load a rom for a gba emulator?

Reply:
though that might need to be rw, mainly for the battery saves. Usually that gets dumped beside the rom. Is that going to be a potential problem?